plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/preferences/PrefPageErrorLogs.java

```diff
@@ -71,6 +75,24 @@ protected Control createPreferenceContent(@NotNull Composite parent) {
                     GeneralUtils.variablePattern(SystemVariablesResolver.VAR_WORKSPACE),
                     GeneralUtils.variablePattern(SystemVariablesResolver.VAR_HOME)));
             logsDebugLocation.setLayoutData(new GridData(GridData.FILL_HORIZONTAL));
+            
+            UIUtils.createControlLabel(groupLogs, CoreMessages.pref_page_logs_files_max_size_label);
+            logFilesMaxSizeSpinner = new Spinner(groupLogs, SWT.BORDER);
+            logFilesMaxSizeSpinner.setDigits(0);
+            logFilesMaxSizeSpinner.setIncrement(10);
+            logFilesMaxSizeSpinner.setMinimum(0);
+            logFilesMaxSizeSpinner.setMaximum(Integer.MAX_VALUE);
+            long bigScriptSize = DBWorkbench.getPlatform().getPreferenceStore().getLong(LogOutputStream.LOGS_MAX_FILE_SIZE);
```

Comment:
Let's extract the common `DBWorkbench.getPlatform().getPreferenceStore()` part and inline these variables in other places